src/app/views/communities/view/overview/edit/edit.ts

```diff
@@ -0,0 +1,201 @@
+import FormCommunityCollaborator from '../../../../../components/forms/community/collaborator/collaborator.vue';
+import FormCommunityTag from '../../../../../components/forms/community/tag/tag.vue';
+import { CommunityThumbnailModal } from '../../../../../components/forms/community/thumbnail/modal/modal.service';
+import { store } from '../../../../../store';
+import { RouteStore, routeStore, RouteStoreModule, RouteStoreName } from './edit.store';
+import AppCommunitiesOverviewEditNotice from './_notice/notice.vue';
+
+const draggable = require('vuedraggable');
+
+type CPayload = {
+	communityPayload: any;
+	collaboratorPayload: any;
+};
+
+@Component({
+	name: 'RouteCommunitiesViewEdit',
+	components: {
+		AppCommunitiesOverviewEditNotice,
+		FormCommunityTag,
+		draggable,
+		AppCardList,
+		AppCardListItem,
+		AppCardListAdd,
+		FormCommunityCollaborator,
+		AppCommunityPerms,
+		AppEditableOverlay,
+		AppCommunityThumbnailImg,
+	},
+	directives: {
+		AppTooltip,
+	},
+})
+@WithRouteStore({
+	store,
+	routeStoreName: RouteStoreName,
+	routeStoreClass: RouteStore,
+})
+@RouteResolver({
+	deps: { params: ['id'] },
+	async resolver({ route }) {
+		const communityPayload = await Api.sendRequest(
+			'/web/dash/communities/' + route.params.id,
+			{}
+		);
+
+		if (communityPayload && communityPayload.community) {
+			const redirect = enforceLocation(route, { path: communityPayload.community.path });
+			if (redirect) {
+				return redirect;
+			}
+		}
+
+		const collaboratorPayload = await Api.sendRequest(
+			'/web/dash/communities/collaborators/' + route.params.id
+		);
+
+		return { communityPayload, collaboratorPayload };
+	},
+	resolveStore({ payload }) {
+		if (payload.communityPayload) {
+			routeStore.commit('populate', payload.communityPayload);
+		}
+	},
+})
+export default class RouteCommunitiesViewEdit extends BaseRouteComponent {
+	@RouteStoreModule.State
+	community!: RouteStore['community'];
+	@RouteStoreModule.State
+	collaboration!: RouteStore['collaboration'];
+
+	collaborators: Collaborator[] = [];
+	activeCollaborator: Collaborator | null = null;
+	isAddingCollaborator = false;
+
+	readonly Collaborator = Collaborator;
+	readonly Screen = Screen;
+
+	get routeTitle() {
+		if (this.community) {
+			return this.$gettextInterpolate(`Edit Community %{ community }`, {
+				community: this.community.name,
+			});
+		}
+		return null;
+	}
+
+	get hasPerms() {
+		return this.community.hasPerms('all');
+	}
+
+	get isOwner() {
+		// The owner's collaboration is not returned from backend.
+		return this.collaboration === null;
+	}
+
+	get shouldShowThumbnail() {
+		return Screen.isXs;
+	}
+
+	routeResolved($payload: CPayload) {
+		if ($payload.collaboratorPayload.success === true) {
+			this.collaborators = Collaborator.populate($payload.collaboratorPayload.collaborators);
+			if (!this.collaborators.length) {
+				this.isAddingCollaborator = true;
+			}
+		}
+	}
+
+	onTagsChange() {
+		this.$emit('tags-change', this.community.tags);
```

Comment:
We don't have a store for view/overview.

Reply:
We planned on moving the edit section to be on the same level as overview, and then a new store would've made more sense. Since this turned out to be a bit more work than initially thought, we're going to skip this for now.